src/applications/simple-forms/21-4142/pages/contactInformation1.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { intersection, pick } from 'lodash';
+import fullSchema from 'vets-json-schema/dist/21-4142-schema.json';
+import addressUiSchema from 'platform/forms-system/src/js/definitions/profileAddress';
+import { veteranFields } from '../definitions/constants';
+
+const { required, properties } = fullSchema.properties[
+  veteranFields.parentObject
+];
+const pageFields = [veteranFields.address];
+
+export default {
+  uiSchema: {
+    'view:title': {
```

Comment:
I think this pattern was recently used in 4555. Overall, for this PR, the code for titles is inconsistent done in different ways, and sometimes adding classNames. So looking for feedback on that if thats ok.